### PR TITLE
Fix internal links and images

### PR DESCRIPTION
**Pull request by Fashander**
# Description

## Readiness checklist

* [ ] I added unit tests for new functionality or bug fixes.
* [ ] I added integration tests for new functionality or bug fixes.
* [ ] I added compatibility tests for new functionality or bug fixes.
* [ ] I made spot refactorings.
* [ ] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by AlekSi — CHANGES_REQUESTED** [review]
This PR adds `.webp` files as blobs in git instead of using Git LFS. Please remove them, apply 9ba1f52f23a90f21d11ec1d4d4671f7640cf4b39, and re-add them

**Review by AlekSi — CHANGES_REQUESTED** [review]
Still the same problem. https://patch-diff.githubusercontent.com/raw/FerretDB/FerretDB/pull/1796.patch, search for "GIT binary patch".

And please remove .DS_Store

**Comment by Fashander** [comment]
@AlekSi Okay. I'm on it,

**Comment by AlekSi** [comment]
@Fashander There are conflicts now

File: website/blog/2022-12-23-ferretdb-new-release-with-tls.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
 slug: ferretdb-new-release-with-tls
 title: "FerretDB v0.7.1 - Now offering basic TLS support"
 author: Alexander Fashakin
+image: ../static//img/blog/TLS.png
```
Comment (AlekSi) [inline website/blog/2022-12-23-ferretdb-new-release-with-tls.md:5]:
```suggestion
image: ../static/img/blog/TLS.png
```

File: website/blog/2022-12-23-ferretdb-new-release-with-tls.md [hunk 2]
```diff
@@ -2,10 +2,12 @@
...
+tags: [release]
 date: 2022-12-23
 ---
 
-![FerretDB 0.7.1 release](https://i0.wp.com/www.swhosting.com/blog/wp-content/uploads/2014/10/TLS.png)
+![FerretDB 0.7.1 release](../static//img/blog/TLS.png)
```
Comment (AlekSi) [inline website/blog/2022-12-23-ferretdb-new-release-with-tls.md:10]:
```suggestion
![FerretDB 0.7.1 release](../static/img/blog/TLS.png)
```

**Review by AlekSi — APPROVED** [review]

**Review by AlekSi — CHANGES_REQUESTED** [review]
CleanShot 2023-01-16 at 15 38 17@2x

Please don't store .webp files with .png extension

**Comment by Fashander** [comment]
@AlekSi Please, I need your help with this. Seems the `.webp` images are still not showing up despite adding it to `git lfs`.

**Comment by Fashander** [comment]
> > Seems the .webp images are still not showing up despite adding it to git lfs.
> 
> It might be GitHub's problem: [github/docs#21798](https://github.com/github/docs/issues/21798)

Oh, that explains it. Thanks. What should I do to get this close to being done though? Should we wait for GitHub to resolve the issue?

**Comment by AlekSi** [comment]
Nothing. The fact that they don't show up in PRs doesn't me they don't work: https://media.githubusercontent.com/media/FerretDB/FerretDB/ee9f09b57be451611efc407bb68665327d860978/website/static/img/blog/4ferrets.webp

**Pull request merged**